### DISCUSSION TITLE
Does COSMOS Support ANSI Escape Sequences?

### DISCUSSION THREAD

**Discussion by ghost** [CosmosOS/Cosmos#1889] [Q&A]
I'm making a library called [CosmosColor](https://github.com/BeriiDevelopment/CosmosColor), but I want to check if CosmosOS' terminal supports ANSI escape sequences. 
E.G. ```\033[30m``` makes the foreground color black.
Is it supported or should I find another way to use ANSI escape sequences, or just give up and use Console.ForegroundColor?

**Comment by quajak** [comment]
AFAIK Cosmos does not support ANSI escape sequences

**Reply by ghost** [reply]
Is there a way to implement support for them?

**Reply by quajak** [reply]
I think so, if you update the code in the Console.Write method to correctly handle them.
